src/test/lambda/local/debugConfiguration.test.ts

```diff
@@ -66,10 +66,7 @@ describe.only('makeCoreCLRDebugConfiguration', async () => {
         return config
     }
 
-    async function makeConfig({
-        codeUri = path.join('foo', 'bar'),
-        port = 42
-    }: {codeUri?:string, port?:number}) {
+    async function makeConfig({ codeUri = path.join('foo', 'bar'), port = 42 }: { codeUri?: string; port?: number }) {
```

Comment:
@hunterwerlla here's a single-line list , prettier did not add a trailing comma.